kythe/proto/generate_protobufs.py

```diff
@@ -35,47 +38,69 @@
 import stat
 import sys
 
+if len(sys.argv) > 2:
```

Comment:
I'd like to put in an appeal for using a flag-parsing library (e.g., [argparse](https://docs.python.org/3/library/argparse.html#module-argparse)) rather than bodging the arguments directly. I realize there's only one right now, but now that we've broken the seal, there will probably be more to come.

Reply:
argparse is in the Python standard library and is available in both Python 3 and Python 2. It is definitely preferable to ad hoc flag parsing.